lib/account.php

```diff
@@ -123,6 +123,7 @@ public function createMailbox($mailBox) {
 		$conn->createMailbox($mailBox);
 		$this->mailboxes = null;
 
+//		$mailBox = \Horde_Imap_Client_Mailbox::get($mailBox, false)->utf7imap;
```

Comment:
I wonder where this is from, as I thought I've removed all utf7imap related code.